code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -109,8 +109,9 @@
 	switch(air.temperature)
 		if(-INFINITY to WATER_VAPOR_DEPOSITION_POINT)
 			if(location?.freeze_turf())
```

Comment:
a: does this mean if we can't freeze the turf we won't react? What does that mean?

Reply:
The reaction isn't really doing anything if that check doesn't pass, so I wouldn't really consider that as reacting.

Reply:
oh TRUE then we don't set the flag
yeah nevermind this is safe